src/components/molecules/card/card.html

```diff
@@ -0,0 +1,12 @@
+<div class="card">
+  <p class="title">card-title</p>
+  <img src="../../../img/card-1.jpg" class="img" />
```

Comment:
```suggestion
<figure class="img-wrapper">
  <img src="../../../img/card-1.jpg" class="img" />
</figure>
```
```css
.figure-wrapper {
  overflow: hidden;
  /** ...その他高さ幅などの指定 **/
}
```

imgを囲むラッパーを作成し、大きさに関するスタイルとoverflowはそこに書くようにするとはみ出さなくなる気がします！